### ISSUE TITLE
[Nova] Using tap-i18n with react?

### ISSUE THREAD

**Issue by Sabst** [TelescopeJS/Telescope#1300]
A more react-compatible approach might be more appropriate unless the issue below is fixed, which is not currently the case:
https://github.com/TAPevents/tap-i18n/issues/115

I understand this is currently working but Blaze should probably stop being pulled by Nova in the future.
What do you think?

**Comment by SachaG** [comment]
Actually the `nova:i18n` package is disabled, so I don't think tap:i18n's Blaze dependency is currently being loaded. I'm happy to explore other i18n solutions, I didn't know tap:i18n was tied in with Blaze. Would you be willing to take the lead on this by any chance?

**Comment by Sabst** [comment]
I followed a path similar to the one described [here](https://medium.com/@tomi.trescak/setting-up-i18n-with-react-intl-in-mantra-react-application-12f90ff34f17#.63pmvqfi1).

I'm currently stuck on the 'require' part (but I do not like it anyway! see my comment on medium) probably because of the package-based architecture of Nova: the file is shipped with the client but I found no way to "import" it so far.

Now the scope of this topic is broader than Nova and it would be nice to see that addressed globally (Meteor Guide?) esp. now we know TAP-i18n is not so appropriate with react (the issue with the meteor guide was raised a few months ago but closed without a visible action).

As a side note, the storage of translated strings in the DB (see current JSON files as fixtures) allows to have a back-office to fine-tune/correct the phrasing or terminology, define new locales...
Does that match with a Nova use-case ? (e.g. _as a Nova admin, I want to change the displayed strings for a given locale online_)
